Formula/manifest-tool.rb

```diff
@@ -0,0 +1,35 @@
+class ManifestTool < Formula
+  desc "Command-line tool to create and query container image manifest list/indexes"
+  homepage "https://github.com/estesp/manifest-tool/"
+  url "https://github.com/estesp/manifest-tool/archive/refs/tags/v2.0.3.tar.gz"
+  sha256 "53bba7cc7465c8347c70e36467848bd385eba2fb51d7aceb269c316382a065a7"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "all"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    package = "busybox:latest"
+    stdout, stderr, = Open3.capture3(
+      bin/"manifest-tool", "inspect",
+      package
+    )
+
+    if stderr.lines.grep(/429 Too Many Requests/).first
+      print "Can't test against docker hub\n"
+      print stderr.lines.join("\n")
+    else
```

Comment:
Can't we specify where to check?

Reply:
Yes... Do you have a repository without a rate limit?

Reply:
How many tests does it actually run? Because everything has _some_ limits

Reply:
One API call.

When I was testing, I made quite a few in a short time and hit the API limit. I don't expect this to normally be hit, but, instead of a flaky test, especially if the previous user of the IP has hit the lookout, I'd rather err on the side of handling the error.

Reply:
Could be hit when we run multiple builds in parallel though. Probably better safe than sorry, especially if this comes up in a long-running PR that tests many formulae.